ConfigurationSystem/Client/Helpers/Registry.py

```diff
@@ -490,3 +490,17 @@ def getUserDict(username):
     if value:
       resDict[key.replace(relPath, '')] = value
   return S_OK(resDict)
+
+
+def getEmailsForGroup(groupName):
+  """ Get email list of users in group
+
+      :param basestring groupName: DIRAC group name
+
+      :return: list
```

Comment:
Following on from below, in that case:
```suggestion
      :return: list of lists -- inner list contains alternative emails for a user
```

Reply:
Done